src/shared/escaping.js

```diff
@@ -19,7 +19,7 @@ export const clientSequences = [
 ]
 
 // sanitizes potentially dangerous characters
-export function escape (info, options, escapeOptions) {
+export function escape (info, options, escapeOptions, escapeKeys) {
```

Comment:
Couldn't `escapeKeys` be a part of `escapeOptions` ?

Reply:
Sure thing :)

Reply:
@pi0 fyi, i reverted back to escapeKeys being a separate arg. The spread syntax added about 0.5KB to the minified build which just isnt worth it imo. Also changed all for...of's as that introduces a Symbol.iterator which needs a polyfill for IE9.